src/wellKnownDidConfiguration/wellKnownDidConfiguration.test.ts

```diff
@@ -58,10 +59,9 @@ describe('Well Known Did Configuration integration test', () => {
     didDocument = await generateDid(account, mnemonic)
 
     didUri = didDocument.uri
-    keyUri = `${didUri}${didDocument.assertionMethod?.[0].id!}`
+    keyUri = `${didUri}${didDocument.assertionMethod![0].id}`
```

Comment:
Would a cleaner approach be to adjust the return type of `generateDid`?

Reply:
We know, but we lied to TS about the return type of `generateDid` and now we have to force it. If we didn’t lie about the return type we would not need `?.` or `!` here.

Reply:
oh I see! Thing is, this is defined on the `DidDocument` type which we import from the sdk and which we'd have to rewrite here for that purpose, all of which seems like a little to much just for a test